Scheduler/FirstComeFirstServe.c

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file FirstComeFirstServe.c
+ */
+
+#include "FirstComeFirstServe.h"
+#include "ScheduleFile.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+void RunProcess (ScheduleData* inputData){
```

Comment:
Rename `RunProcess` to `RunFCFSProcess` or `RunFCFS` or something along those lines. This is for the same reasoning as #4  where I asked Jasper to change the name of his RoundRobin function.

Reply:
Changed to RunFCFSProcess in next commit